apps/gro/fields/index.js

```diff
@@ -87,6 +87,19 @@ module.exports = {
     mixin: 'input-text',
     validate: ['required', 'email']
   },
+
+  'confirm-email': {
+    mixin: 'radio-group',
+    validate: ['required'],
+    legend: {
+      className: 'visuallyhidden'
```

Comment:
is this an existing css class name? if not, just call the class hidden, this is overly specific

Reply:
This is what is used for Hof components. The GDS class is  `govuk-visually-hidden` .